src/config.ts

```diff
@@ -54,6 +54,9 @@ export interface AppConfig {
     policy: string;
     writeScope: string;
     readScope: string;
+    egoRootRest: string;
+    egoScoreProxyClientId: string;
+    egoScoreProxyClientSecret: string;
```

Comment:
lets drop the `scoreProxy` part of the name, now that we know it doesnt use the score-proxy permissions.